### ISSUE TITLE
`trio200-blocking-calls =` is loaded but not parsed from a config file

### ISSUE THREAD

**Issue by Zac-HD** [Zac-HD/flake8-trio#84]
I get an error on line 1353 because it hasn't been parsed from list-of-strings into a dict:

https://github.com/Zac-HD/flake8-trio/blob/7cdd9180e3aefca89d1a58d197022f266b45868c/flake8_trio.py#L1350-L1356

I also started on a quick regression test, but don't have time to finish it right now: https://github.com/Zac-HD/flake8-trio/compare/config-error

**Comment by jakkdl** [comment]
I think the actual problem here is flake8's `comma_separated_list` .. apparently treating spaces as commas?
```bash
flake8 --no-checkpoint-warning-decorators="a b, c d" <file>
``` 
```python
ipdb> self.options.no_checkpoint_warning_decorators
['a', 'b', 'c', 'd']
```

**Comment by jakkdl** [comment]
New try: It looks like the `action` parameter to `flake8.options.manager.OptionManager.add_option()` is entirely ignored when parsing a config file. The supplied function is never called, and it's the same thing with `ListOfIdentifiers` for `--startable-in-context-manager`, invalid identifiers aren't flagged. Instead the values are just accepted straight up.
Passing the same values at the command line works as expected.

Will delve more into flake8's internals to figure out what's going on, and probs open an issue over there. I'm opening a PR with a dirty workaround, travelling to family for holidays in the coming days so a more thorough exploration might be delayed.

(and then also fix spaces)

**Comment by jakkdl** [comment]
Opened https://github.com/PyCQA/flake8/issues/1770 and https://github.com/PyCQA/flake8/issues/1771

**Comment by Zac-HD** [comment]
Apparently we should just avoid `action` entirely and use `type` instead https://github.com/PyCQA/flake8/issues/1770#issuecomment-1358067558 🤷
